spec/unit/http/client_spec.rb

```diff
@@ -376,14 +376,14 @@ def expect_http_error(cause, expected_message)
 
     it 'raises an ArgumentError if `body` is missing from the options hash' do
       expect {
-        client.post(uri, options: {content_type: 'text/plain'})
+        client.post(uri, headers: {'Content-Type' => 'text/plain'})
       }.to raise_error(ArgumentError, /'post' requires a 'body' option/)
     end
 
-    it 'raises an ArgumentError if `content_type` is missing from the options hash' do
+    it 'raises an ArgumentError if `content_type` is missing from the options header' do
```

Comment:
`header hash`? `header options`? `options header` feels strange imo, but doesn't really matter that much

Reply:
yeah agree, how about `headers hash`, as the keyword is "headers"?